doc/source/tutorials/quickstart.rst

```diff
@@ -357,84 +374,239 @@ Step 6: Now to make it turn
+Well, a true and proper gravity turn is a very complex bit of math that is best left as an exercise for the reader, given that the goal of **kOS** is to let you write your OWN autopilot, not to write it for you. But to give some basic examples of commands, lets just make a crude gravity turn approximation that simply flies the ship like a lot of new **KSP** pilots learn to do it for the first time:
 
-- Fly straight up to 10000m.
-- Aim at 45 degrees down toward the east until 40000m.
-- Thrust horizontally east after that.
+- Fly straight up until your velocity is 100m/s.
+- Pitch ten degrees towards the East.
+- Continue to pitch 10 degrees down for each 100m/s of velocity.
 
 To make this work, we introduce a new way to make a Direction, called the HEADING function. Whenever you call the function HEADING(a,b), it makes a Direction oriented as follows on the navball:
 
 - Point at the compass heading A.
 - Pitch up a number of degrees from the horizon = to B.
 
-So for example, HEADING(45,10) would aim northeast, 10 degrees above the horizon. Combining this with the WHEN command from before, we get this section::
+So for example, HEADING(45,10) would aim northeast, 10 degrees above the horizon. We can use this to easily set our orientation. For example::
 
-    WHEN SHIP:ALTITUDE > 10000 THEN {
-        PRINT "Starting turn.  Aiming to 45 degree pitch.".
-        LOCK STEERING TO HEADING(90,45). // east, 45 degrees pitch.
-    }
-    WHEN SHIP:ALTITUDE > 40000 THEN {
-        PRINT "Starting flat part.  Aiming to horizon.".
-        LOCK STEERING TO HEADING(90,0). // east, horizontal.
-    }
+    //This locks our steering to due east, pitched 45 degrees above the horizon.
+    LOCK STEERING TO HEADING(90,45).
 
-Note that these lack the command PRESERVE like the previous WHEN example had. This is because we want these to trigger just once and then never again. There's no point in constantly telling **kOS** to reset the steering to the same thing over and over as the script runs.
+Instead of using WAIT UNTIL to pause the script and keep it from exiting, we can use an UNTIL loop to constantly perform actions until a certain condition is met. For example::
 
-Now, if you insert this new section to the script, we have a nice nifty example of a start of a launching script. Note that it works even if you insert it at the top of the script, because it sets up the triggers to occur LATER when the condition becomes true. They don't execute right away::
+    UNTIL APOAPSIS > 100000 {
+        LOCK STEERING TO HEADING(90,90). //90 degrees east and pitched up 90 degrees (straight up)
+        PRINT SHIP:APOAPSIS AT (0,15).
+        //We use the PRINT AT() command here to keep from printing the same thing over and
+        //over on a new line every time the loop iterates. Instead, this will always print 
+        //the apoapsis at the same point on the screen.
```

Comment:
Hmm.... maybe two commands back to back:
```
    PRINT "            " at (0,15).
    PRINT ROUND(SHIP:APOAPSIS,0) at (0,15).
````
Just because one of the frequent problems new users encounter when using this print technique is how to deal with a number that gets smaller over time.  In this case it doesn't, but if it does, then you get leftover digits on the screen to the right that falsify the display and make the number look bigger than it is.  This isn't *too* complicated and it lets us introduce the concept.